packages/shared/src/components/cards/v1/ActionButtons.tsx

```diff
@@ -0,0 +1,150 @@
+import React, { ReactElement } from 'react';
+import classNames from 'classnames';
+import dynamic from 'next/dynamic';
+import { Post, UserPostVote } from '../../../graphql/posts';
+import InteractionCounter from '../../InteractionCounter';
+import UpvoteIcon from '../../icons/Upvote';
+import CommentIcon from '../../icons/Discuss';
+import { Button, ButtonColor, ButtonVariant } from '../../buttons/ButtonV2';
+import { SimpleTooltip } from '../../tooltips/SimpleTooltip';
+import { useFeedPreviewMode } from '../../../hooks';
+import BookmarkIcon from '../../icons/Bookmark';
+import DownvoteIcon from '../../icons/Downvote';
+
+const ShareIcon = dynamic(
+  () => import(/* webpackChunkName: "shareIcon" */ '../../icons/Share'),
+);
+
+export interface ActionButtonsProps {
+  post: Post;
+  onMenuClick?: (e: React.MouseEvent) => unknown;
+  onUpvoteClick?: (post: Post) => unknown;
+  onDownvoteClick?: (post: Post) => unknown;
+  onCommentClick?: (post: Post) => unknown;
+  onBookmarkClick?: (post: Post) => unknown;
+  onShareClick?: (event: React.MouseEvent, post: Post) => unknown;
+  onReadArticleClick?: (e: React.MouseEvent) => unknown;
+  className?: string;
+  insaneMode?: boolean;
+  openNewTab?: boolean;
+}
+
+type ShareButtonProps = {
+  onShareClick?: (event: React.MouseEvent, post: Post) => unknown;
+  post: Post;
+};
```

Comment:
Just converting the new ones.
```suggestion
interface ShareButtonProps {
  onShareClick?: (event: React.MouseEvent, post: Post) => unknown;
  post: Post;
}

```